pokemongo_bot/__init__.py

```diff
@@ -341,18 +341,19 @@ def _set_starting_position(self):
                 if not self.config.location:
                     sys.exit("No cached Location. Please specify initial location.")
                 else:
-                    pass
+                    self._read_config_location()
```

Comment:
You can remove the else and read_config_location here, then move the other read_config_location outside the else.